src/tools/miri/src/helpers.rs

```diff
@@ -151,12 +151,14 @@ pub fn iter_exported_symbols<'tcx>(
     let dependency_format = dependency_formats
         .get(&CrateType::Executable)
         .expect("interpreting a non-executable crate");
-    for cnum in dependency_format.iter().enumerate().filter_map(|(num, &linkage)| {
-        // We add 1 to the number because that's what rustc also does everywhere it
-        // calls `CrateNum::new`...
+    for cnum in dependency_format.iter_enumerated().filter_map(|(num, &linkage)| {
         #[expect(clippy::arithmetic_side_effects)]
```

Comment:
There's no arithmetic here any more, so this can be removed

Reply:
Removed